terraform/azure/security-groups.tf

```diff
@@ -0,0 +1,32 @@
+/* Default security group */
+resource "azure_security_group" "default" {
+  name     = "default-apollo-mesos"
+  location = "${var.region}"
+}
+
+resource "azure_security_group_rule" "all-inbound" {
```

Comment:
There seems to be some special constants described here - https://azure.microsoft.com/en-us/documentation/articles/virtual-networks-nsg/ such as INTERNET / VIRTUAL_NETWORK - might be worth double checking the docs